### PR TITLE
Added a contact page and terms & condition page

### LINKED ISSUE #165: [Feature] Creating contact us page.
### Description

- To create a "Contact Us" page that complements the overall theme and styling of your website.
- I will ensure the design is cohesive with your website's color scheme, typography, and layout to maintain brand consistency.
- The link to the contact us page will be provided in the footer.

I want to work on this issue kindly assign it to me under GSSOC'24.

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/pooranjoyb/popShop/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/pooranjoyb/popShop/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by gdhyani**
Created Contact Us and Terms and Condition Page. Also did a little change on Footer to add links of these pages.

Fixes #165 and #174

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor Code
- [ ] A documentation update
- [ ] Others(mentioned in the issue number)

# How Has This Been Tested?

Just created 2 pages - Contact Us and Terms and Condition Page

# Screenshorts and Vedios:

image

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pooranjoyb — COMMENTED** [review]
Great job, @gdhyani! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

File: src/pages/Contact/Contact.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            {/* main */}
+            <div className="mx-auto max-w-screen-xl flex px-4 pt-8 sm:pt-8 pb-4 ">
+                <Head h1="Contact" h2="Us" />
+            </div>
+            <h1 className="text-base md:text-lg font-medium text-[#073b4c] mx-auto max-w-screen-xl px-4 ">
+                Need to get in touch with us? Either fill out the form with your
+                inquiry or find the department email you'd like to contact
+                below.
+            </h1>
+            <div className="px-4 mt-10 mb-20 md:mb-40 sm:py-6 mx-auto max-w-screen-md lg:py-4 md:px-6 flex flex-col gap-4">
```
Comment (pooranjoyb) [inline src/pages/Contact/Contact.tsx:31]: The `<Head>` component should be inside the this div. 
I mean the `Form` and The heading should have the same alignment.

File: src/pages/T&C/TermsCond.tsx
Comment (pooranjoyb) [inline src/pages/T&C/TermsCond.tsx:1]: Add the `Navbar` and `Footer` component. 
Also use the `Head` component for the Headings. Donot use manual texts.

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: src/pages/T&C/TermsCond.tsx
Comment (gdhyani) [reply src/pages/T&C/TermsCond.tsx:1]: Although I have `Head` as the primary heading, I need small text for the other headline, and the `head` component has a fixed font size, so I followed your previous approach on the home page, borrowed the style, and created the new headline.
Now, coming to the alignment, I have made changes for that. I will send you a PR in a while.

File: src/pages/T&C/TermsCond.tsx [hunk 2]
```diff
@@ -0,0 +1,371 @@
+function TermsCond() {
+    return (
```
Comment (pooranjoyb) [inline src/pages/T&C/TermsCond.tsx:2]: Add `Navbar` here and `Footer` Below. Rest everything looks good for the merge. Do the changes and let us know, we'll merge. @gdhyani

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

**Comment by gdhyani** [comment]
Done!

**Review by pooranjoyb — APPROVED** [review]

**Pull request merged**